Scripts/Common/Functions.sh

```diff
@@ -23,7 +23,7 @@ export -f startPatcher;
 
 resetWorkspace() {
 	umask 0022;
-	repo forall -c 'git add -A && git reset --hard' && rm -rf out DOS_PATCHED_FLAG && repo sync -j8 --force-sync --detach;
+	repo forall -j${DOS_MAX_THREADS_REPO} -c 'git add -A && git reset --hard' && rm -rf out DOS_PATCHED_FLAG && repo sync -j${DOS_MAX_THREADS_REPO} --force-sync --detach;
```

Comment:
using too many threads on sync will get you rate limited by google and temporarily blocked

using many threads on reset may cause thrashing, and already completes quickly

Reply:
That's why there's a max allowed value. Again this is more to limit (e.g using 1 helps for debug but using other values can help speeding up the process - yes even for reset)